recipes/qtxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os, glob, shutil
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conans import ConanFile, tools
+
+
+class QtXlsxWriterConan(ConanFile):
+    name = "qtxlsxwriter"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dbzhang800/QtXlsxWriter"
+    description = "A library that can read and write Excel files"
+    topics = ("qtxlsxwriter", "excel", "xlsx", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps"
+    exports_sources = [
+        "CMakeLists.txt",
+        os.path.join("cmake", "*"),
+        os.path.join("patches", "*"),
+        os.path.join("src", "CMakeLists.txt")
```

Comment:
typically we keep all the CMake in one file. Could you please combine them and keep the root one... we call this our "camke_wapper" and there are a few example of these

Reply:
Fixed by [4bc210b](https://github.com/conan-io/conan-center-index/pull/4838/commits/4bc210b8e32c62e9235ed657472a37fd130a559c)